Buildings/Electrical/AC/ThreePhasesBalanced/Loads/MotorDrive/Coupled/Chiller.mo

```diff
@@ -0,0 +1,284 @@
+    "Temperature difference between refrigerant and working fluid outlet in evaporator"
+    annotation (Dialog(group="Efficiency"));
+
+  //Motor parameters
+  parameter Integer pole=4 "Number of pole pairs";
+  parameter Integer n=3 "Number of phases";
+  parameter Modelica.Units.SI.Resistance R_s=0.641
+    "Electric resistance of stator";
+  parameter Modelica.Units.SI.Resistance R_r=0.332
+    "Electric resistance of rotor";
+  parameter Modelica.Units.SI.Reactance X_s=1.106
+    "Complex component of the impedance of stator";
+  parameter Modelica.Units.SI.Reactance X_r=0.464
+    "Complex component of the impedance of rotor";
+  parameter Modelica.Units.SI.Reactance X_m=26.3
+    "Complex component of the magnetizing reactance";
+  parameter Modelica.Units.SI.Inertia JLoad(min=0)=2 "Load inertia";
+  parameter Modelica.Units.SI.Inertia JMotor=2         "Motor inertia";
+
+  ThermoFluid.Chiller mecChi(
+    redeclare final package Medium1 = Medium1,
+    redeclare final package Medium2 = Medium2,
+    final m1_flow_nominal=m1_flow_nominal,
+    final m2_flow_nominal=m2_flow_nominal,
+    final dTEva_nominal=dTEva_nominal,
+    final dTCon_nominal=dTCon_nominal,
+    final use_eta_Carnot_nominal=use_eta_Carnot_nominal,
+    final etaCarnot_nominal=etaCarnot_nominal,
+    final COP_nominal=COP_nominal,
+    final TCon_nominal=TCon_nominal,
+    final TEva_nominal=TEva_nominal,
+    final a=a,
+    final dp1_nominal=dp1_nominal,
+    final dp2_nominal=dp2_nominal,
+    final TAppCon_nominal=TAppCon_nominal,
+    final TAppEva_nominal=TAppEva_nominal,
+    final P_nominal=P_nominal,
+    final QEva_flow_nominal=QEva_flow_nominal,
+    final QCon_flow_nominal=QCon_flow_nominal,
+    final Nrpm_nominal=Nrpm_nominal) "Chiller model with mechanical interface"
+    annotation (Placement(transformation(extent={{-10,-10},{10,10}})));
+  final Modelica.Blocks.Sources.RealExpression loaTor(y=mecChi.shaft.tau) "Chiller torque block"
+    annotation (Placement(transformation(extent={{0,40},{-20,60}})));
+  InductionMotors.SquirrelCageDrive simMot(
+    final pole=pole,
+    final n=n,
+    final J=JMotor,
+    final R_s=R_s,
+    final R_r=R_r,
+    final X_s=X_s,
+    final X_r=X_r,
+    final X_m=X_m,
+    final VFD(reverseActing=false)) "Motor model"
+    annotation (Placement(transformation(extent={{-40,70},{-20,90}})));
+
+  Modelica.Blocks.Interfaces.RealInput setPoi "Set point of control target"
+    annotation (Placement(transformation(
+        extent={{-10,-10},{10,10}},
+        rotation=0,
+        origin={-110,90}), iconTransformation(
+        extent={{-10,-10},{10,10}},
+        rotation=0,
+        origin={-110,90})));
+  Modelica.Blocks.Interfaces.RealInput meaPoi "Measured value of control target"
+    annotation (Placement(transformation(extent={{-120,20},{-100,40}}),
+        iconTransformation(extent={{-120,20},{-100,40}})));
+  Modelica.Blocks.Interfaces.RealOutput P(final quantity="Power",final unit="W")
```

Comment:
improve the text format

Reply:
Thank you for viewing, I've corrected the text format of all files to make sure they match the other models in MBL.